drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/domelements/ScenarioCellTextAreaDOMElement.java

```diff
@@ -115,7 +115,9 @@ protected void internalFlush(final String value) {
 
     @Override
     public void detach() {
-        flush(getValue());
         super.detach();
+        if (scenarioGridCell != null) {
+            scenarioGridCell.setEditingMode(false);
```

Comment:
What is the reason of this change?

Reply:
@danielezonca Because I found a bug during my manual testing. Basically, when a cell value is edited, a double command is created (one calling `flush()` and the second one calling `detach()` . I aware the best way to manage it should be to create a new PR, but I preferred to include it in this PR in order to include it to DevPreview cut off (1th July).